blog-service/2024-10-11-collection.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Automox C2C Source (Collection)
```

Comment:
You need to remove this release note file dated 10-11. The release note was already published on 10-14: https://help.sumologic.com/release-notes-service/2024/10/14/collection/

Reply:
whoa, great catch! It must've gotten picked up in the merge with main